scripts/requirements.txt

```diff
@@ -1,6 +1,8 @@
-recommonmark==0.4.0
-sphinxcontrib-mscgen>=0.5
-imagesize>=1.2.0
 ecdsa
+editdistance>=0.5.0
```

Comment:
This should not have been merged. Please put it back in requirements-extra.txt along with pygit2. Those requirements are for internal use west extensions.